freqtrade/optimize/backtesting.py

```diff
@@ -255,7 +255,7 @@ def _get_sell_trade_entry(self, trade: LocalTrade, sell_row: Tuple) -> Optional[
 
         if sell.sell_flag:
             trade.close_date = sell_row[DATE_IDX]
-            trade.sell_reason = sell.sell_type.value
+            trade.sell_reason = sell.sell_reason or sell.sell_type.value
```

Comment:
i guess we can now use `sell.sell_reason` directly, as the `or` is already happening in SellCheckType ...